### PR TITLE
Add general support for adding additional auth params to OIDC connector

### PR DESCRIPTION
**Pull request by roddyherries**

### CODE REVIEW

File: connector/oidc/oidc_test.go [hunk 1]
```diff
@@ -271,6 +274,25 @@ func TestHandleCallback(t *testing.T) {
 				"cognito:groups": []string{"group3", "group4"},
 			},
 		},
+		{
```
Comment (alextechamigo) [inline connector/oidc/oidc_test.go]: this looks identical to the previous test?

**Review by alextechamigo — COMMENTED** [review]
Looks ok, but a couple things I spotted/wondered:
 * the TestHandleCallback tests seem to have been altered but on visual inspection there's no new test or material change?
 * When a user sets additionalAuthRequestParams it looks like it will override existing parameters(? depending on how the opts are ultimately processed). I suppose this is user/configuration error that could be highlighted in docs, as it's possible to establish conflicting configurations that way.

File: connector/oidc/oidc_test.go (hunk 1 above)
Comment (roddyherries) [reply connector/oidc/oidc_test.go]: correct, archaeological artefact, have removed it.

**Comment by roddyherries** [comment]
Hey @alextechamigo  - additional auth params are added to the url.Values map. It would be possible to override spec  params added by the OIDC connector. My opinion is that you would use a field called AdditionalParams... as an advanced capability. Your suggestion of warning in docs is a good one <-- added

**Comment by alextechamigo** [comment]
looks good to me.

**Pull request merged**